spec/lib/html_processors/amp_img_spec.rb

```diff
@@ -0,0 +1,117 @@
+require 'html_processor'
+require 'html_processor/amp_img'
+
+RSpec.describe HTMLProcessor::AmpImg do
+  let(:processor) { described_class.new(html) }
+
+  describe '.process' do
+    let(:parent_container) { Nokogiri::XML(processed_html.strip).children.first }
+    let(:amp_img_tag) { parent_container.children.first }
+    let(:noscript_tag) { amp_img_tag.children.first }
+    let(:attributes) { subject.attributes.transform_values!(&:value) }
+    let(:processed_html) { processor.process(HTMLProcessor::IMG) }
```

Comment:
can you rename to subject, please? It makes easier to understand what is the one important to follow through the specs.